install.yml

```diff
@@ -1,10 +1,11 @@
 ---
 - name: Generate full namespace
   ansible.builtin.set_fact:
-    install_namespace: "opcap-{{ namespace }}-{{ item | lower }}"
+    install_namespace: >
+      "opcap-{{ manifest_data.packageName | replace('.', '-') | lower | sha1 | truncate(8, False, '', 0) }}-{{ item | lower | truncate(1, False, '', 0) }}"
```

Comment:
I think the `opcap-` is superfluous. Let's just get rid of it. 

The `replace` is no longer necessary since we aren't using it directly.

Lower is probably unnecessary as well for the packagename part. 

For the second part, I would swap the truncate and the lower so that it is only running lower on a single char.

Up to you if it makes it more readable to use join.